vllm/model_executor/layers/spec_decode_base_sampler.py

```diff
@@ -5,6 +5,8 @@
 import torch.jit
 import torch.nn as nn
 
+Device = Union[int, str]
```

Comment:
We already have vllm.utils.Device. Better to use another name or just not define it.

Reply:
I said earlier to move it to a separate one to make the code have less boilerplate 😆 . will defer to your best judgement @LiuXiaoxuanPKU .